File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int org = x;
        int rev = 0;
        int a;
        while(x>0){
            a = x%10;
            rev= rev*10+a;
            x/=10;
        }
        if(rev == org){
            return true;
        }
        else{
            return false;
        }
    }
}

File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let n = x, sum = 0, rem = 0;
    while(n>0){
        rem = n%10;
        sum = sum*10 + rem;
        n = Math.floor(n/10);
    }
    return sum == x;
};